src/test/java/hudson/plugins/jira/pipeline/JiraJobActionTest.java

```diff
@@ -0,0 +1,43 @@
+package hudson.plugins.jira.pipeline;
+
+import hudson.plugins.jira.JiraSite;
+import hudson.plugins.jira.model.JiraIssue;
+import jenkins.branch.MultiBranchProject;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import java.net.URL;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class JiraJobActionTest {
+
+    @Test
+    public void testDetectBranchNameIssue() throws Exception {
+        JiraSite site = mock(JiraSite.class);
+        WorkflowJob job = mock(WorkflowJob.class);
```

Comment:
@jglick any idea on how to mock a WorkflowJob? Mockito won't mock finals and I don't think setting up a full multibranch pipeline backed by a git repo, scanning, etc would be appropriate here.